### ISSUE TITLE
Memories/x years since feature plays sound twice

### ISSUE THREAD

**Issue by jazzdd86** [immich-app/immich#7562]
### The bug

If you click on the memories tile in the timeline including videos you can hear the sound twice (mobile app). Web App works ok.
When you close the memory function, you can still hear the sound of the video - but only once.
You need to close the app to stop hearing the videos audio.

### The OS that Immich Server is running on

Docker on Arch Linux

### Version of Immich Server

1.97.0

### Version of Immich Mobile App

1.97.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
default compose file
```

### Your .env content

```Shell
default env file - nothing special
```

### Reproduction steps

```bash
1. Click on the Memories function with videos
2.
3.
...
```

**Comment by alextran1502** [comment]
I, too experienced this a few times, I think something doesn't expose correctly with the new hook mechanism for video players cc @martyfuhry

**Comment by martyfuhry** [comment]
Thanks for the report. I'll take a look! I noticed another issue recently in video memories, so I'll be looking through all of this soon.

**Comment by aveexy** [comment]
Only happens on Android for me, on iOS it works.

**Comment by alextran1502** [comment]
Fixed
